plugins/uefi/fu-plugin-uefi.c

```diff
@@ -510,6 +510,10 @@ fu_plugin_uefi_coldplug_device (FuPlugin *plugin, FuUefiDevice *dev, GError **er
 	if (!fu_device_probe (FU_DEVICE (dev), error))
 		return FALSE;
 
+	/* set this flag for all Lenovo hardware */
```

Comment:
This can't be done in a `.quirk` file?
Is there any other criteria that can be used to identify this?  Versions < `X.Y.Z` or something?

Reply:
It at least _seems_ like it should be a fixed list of systems that should be affected to me, and known when those systems no longer need this behavior and can get on the "regular" train.

Reply:
I've asked Lenovo for this, lets see what they come up with.